<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GNMViewDialogBase</class>
 <widget class="QDialog" name="GNMViewDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load GNM network</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabwMain">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabFile">
      <attribute name="title">
       <string>Directory</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="editFile"/>
         </item>
         <item>
          <widget class="QPushButton" name="butFile">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="butRead">
     <property name="text">
      <string>Read network</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="teditMain">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="butLoad">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Load to project</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
